vips/vip-128/vip-128-testnet.ts

```diff
@@ -0,0 +1,184 @@
+      Set Collateral Factor to 55% in the OLD TUSD market
+      Add the market for (new) TUSD to the Comptroller
+      Comptroller._supportMarket
+      Configure the rest of parameters in the (new) TUSD market:
+      Comptroller._setMarketSupplyCaps
+      Comptroller._setMarketBorrowCaps
+      Comptroller._setVenusSpeeds
+      Mint an initial amount of VTUSD tokens in the new market
+      Accept the admin of the VTUSD market
+      Configure the price feed in the ResilientOracle for the new VToken contract (previously deployed), using the same configuration that we have for TUSD:
+      Type of oracle: Chainlink
+      Feed: 0xa3334a9762090e827413a7495afece76f41dfc06
+      Max stale period: 24 hours
+    
+    Parameters:
+      New VTUSD market:
+      Collateral Factor: 0%
+      Reserve Factor: 10%
+      Borrow Cap: 600,000 TUSD
+      Supply Cap: 1,000,000 TUSD
+      OLD VTUSD market:
+      Collateral Factor: 55%
+      Reserve factor: 100%
+      New symbol: vTUSDOLD
+      New name: Venus TUSDOLD
+    `,
+    forDescription: "I agree that Venus Protocol should proceed with the TUSD contract migration",
+    againstDescription: "I do not think that Venus Protocol should proceed with the TUSD contract migration",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with the TUSD contract migration or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: COMPTROLLER,
+        signature: "_setActionsPaused(address[],uint8[],bool)",
+        params: [[OLD_VTUSD], [Actions.MINT, Actions.BORROW, Actions.ENTER_MARKETS], true],
+      },
+      {
+        target: OLD_VTUSD,
+        signature: "_setImplementation(address,bool,bytes)",
+        params: [VTUSD_RESETTER, false, "0x"],
+      },
+      {
+        target: OLD_VTUSD,
+        signature: "_setImplementation(address,bool,bytes)",
+        params: [VTOKEN_IMPLEMENTATION, false, "0x"],
+      },
+
+      {
+        target: OLD_VTUSD,
+        signature: "_setReserveFactor(uint256)",
+        params: [parseUnits("1", 18)],
+      },
+
+      {
+        target: COMPTROLLER,
+        signature: "_setCollateralFactor(address,uint256)",
+        params: [OLD_VTUSD, parseUnits("0.55", 18)],
+      },
+
+      {
+        target: COMPTROLLER,
+        signature: "_supportMarket(address)",
+        params: [NEW_VTUSD],
+      },
+
+      {
+        target: COMPTROLLER,
+        signature: "_setMarketSupplyCaps(address[],uint256[])",
+        params: [[NEW_VTUSD], [parseUnits("1000000", 18)]],
+      },
+
+      {
+        target: COMPTROLLER,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[NEW_VTUSD], [parseUnits("600000", 18)]],
+      },
+
+      {
+        target: COMPTROLLER,
+        signature: "_setVenusSpeeds(address[],uint256[],uint256[])",
+        params: [
+          [OLD_VTUSD, NEW_VTUSD],
+          ["0", "868055555555556"],
+          ["0", "868055555555556"],
```

Comment:
I would set the same rewards (6.25 XVS/day) that we are setting in main net
```suggestion
          ["0", "217013888888889"],
          ["0", "217013888888889"],
```